### PR TITLE
Checking things for the preview

### PR DESCRIPTION
**Pull request by roshnaeem**
**Description:**

Added `kde` in the https://github.com/arviz-devs/arviz/pull/1818 but needed to fix the order of list. 

1. Fixed the alphabetical order
2. Added empty lines between the text and `.. plot::` directive.

### CODE REVIEW

**Comment by aloctavodia** [comment]
@roshnaeem Is this still a draft?

**Comment by roshnaeem** [comment]
@OriolAbril moved it to draft, he might have some idea to modify it.

File: arviz/stats/density_utils.py [hunk 1]
```diff
@@ -436,6 +440,7 @@ def kde(x, circular=False, **kwargs):
         >>> plt.show()
 
     Default density estimation for linear data with custom limits
+
     .. plot::
         :context: close-figs
     # It accepts tuples and lists of length 2.
```
Comment (OriolAbril) [inline arviz/stats/density_utils.py:446]: This isn't rendering correctly. All directives need a blank line before and after, and the comment should be indented as it should be treated as code.

File: arviz/stats/density_utils.py [hunk 2]
```diff
@@ -436,14 +440,18 @@ def kde(x, circular=False, **kwargs):
         >>> plt.show()
 
     Default density estimation for linear data with custom limits
+
     .. plot::
         :context: close-figs
-    # It accepts tuples and lists of length 2.
+
+        # It accepts tuples and lists of length 2.
+
```
Comment (OriolAbril) [inline arviz/stats/density_utils.py:448]:
```suggestion
        # It accepts tuples and lists of length 2.
```

Comment (roshnaeem) [reply arviz/stats/density_utils.py]: Done

**Pull request closed without merging**